Tools/AnnotationTool/AnnotationTool.cs

```diff
@@ -313,7 +313,7 @@ void HandleBrushSize(float value)
 		if (m_AnnotationPointer != null)
 		{
 			var brushSize = m_Preferences.brushSize;
-			if (VRSettings.loadedDeviceName == "OpenVR") // For vive controllers, use 1:1 touchpad setting.
+			if (UnityEngine.XR.XRSettings.loadedDeviceName == "OpenVR") // For vive controllers, use 1:1 touchpad setting.
```

Comment:
This should have been fixed before AnnotationTool landed (my bad) but I'm noticing it here.

This should use IUsesProxyType, rather than checking the device name.